src/pages/en/getting-started.md

```diff
@@ -8,6 +8,10 @@ description: A basic intro to Astro.
 ---
 Static Site Generator  🚀  Bring your own Framework  🚀  Ship Less JavaScript
```

Comment:
No longer just a static site generator! Not sure what should go here instead, probably just a 1-2 sentence description of Astro?

Reply:
Working on this now for our own Repo readme, I'll copy something here when I have it. 

Also, Feel free to merge this without the change and I can create a PR for it later.